src/components/Sorting.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="flex w-full justify-between flex-wrap gap-5">
+    <div class="flex gap-3">
+      <button
+        @click="handleClick('title')"
+        :class="{
+          'sort-buttons': true,
+          ascendant: sortBy === 'title' && sortDirection === 'asc',
```

Comment:
Please create const variables for 'title', 'text' and 'priority'. These strings are used more times in this file.

Reply:
Also please create variables for the following conditions because those are checked more times in this file as well.
sortBy === 'title'
sortBy === 'text'
sortBy === 'priority'